### DISCUSSION TITLE
Spotilife vs Sposify

### DISCUSSION THREAD

**Discussion by bachig26** [SpotCompiled/SpotilifeC#7] [Q&A] [answered]
@yodaluca23 can you give brief information and comparison to the both app.
I looked through everywhere possible, but I couldn't get a good, well listed difference between them.

also, if possible please let me know if there is any discord server or contact available to reach you.

**Comment by bachig26** [comment]
@yodaluca23 what do you think I use between [SpotilifeC](https://github.com/SpotCompiled/SpotilifeC) and [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus) if I want to use one app, I'm really confused.

**Reply by yodaluca23** [reply]
I would recommend [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus) if you don't know which one to use.

**Comment by yodaluca23** [comment, marked as answer]
[Spotilife](https://julio.hackyouriphone.org/description.html?id=com.julioverne.spotilife), pretty much just adds Spotify Premium features for free, it is included in [SpotilifeC](https://github.com/SpotCompiled/SpotilifeC) and [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus).

[Sposify](https://repo.dynastic.co/com.spos) adds no paid features, it simply adds a bunch of other QOL features to Spotify, and it is included [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus), I'm working on creating [SpotsifyC](https://github.com/SpotCompiled/SposifyC), which will have just Sposify.

TLDR:
[Spotilife](https://julio.hackyouriphone.org/description.html?id=com.julioverne.spotilife) (included with [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus) and [SpotilifeC](https://github.com/SpotCompiled/SpotilifeC)) = free Spotify Premium.
[Sposify](https://repo.dynastic.co/com.spos) (included in [SpotC++](https://github.com/SpotCompiled/SpotC-Plus-Plus)) = QOL enhancements.
